files/brews/hub.rb

```diff
@@ -0,0 +1,26 @@
+require 'formula'
+
+class Hub < Formula
+  homepage "http://hub.github.com/"
+
+  if RUBY_PLATFORM =~ /x86_64/
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_amd64_v2.2.0-rc1.tar.gz"
+    sha1 "413ce80d32f6313cef010e0edb1e664e1fff7427"
+  else
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_386_v2.2.0-rc1.tar.gz"
+    sha1 "059e6d1759b568fd79bacfc397de199135056ff1"
+  end
+
+  def install
+    bin.install "hub"
+    man1.install Dir["man/*"]
+    bash_completion.install "etc/hub.bash_completion.sh"
+    zsh_completion.install "etc/hub.zsh_completion" => "_hub"
+  end
+
+  test do
+    HOMEBREW_REPOSITORY.cd do
+      assert_equal "bin/brew", shell_output("#{bin}/hub ls-files -- bin").strip
```

Comment:
This test relies on the fact that currently the only file checked in Homebrew's `bin` directory is `brew`. This can change in the future.

How about calling `hub version` instead and verifying that "hub" appears in the output? We could also verify the exact version number, because this formula doesn't seem to support `--HEAD` installs and therefore the version output will always match the tagged version that's hardcoded in this formula.